### ISSUE TITLE
Orange Windows Installer publisher unknown

### ISSUE THREAD

**Issue by jerneju** [biolab/orange3#2439]
##### Orange version
Orange Windows installer.
_Orange3-3.4.4.win32-py3.4-install.exe_

##### Expected behavior
Publisher should be written (see below where).

##### Actual behavior
![windows_installation_unknown_publisher](https://user-images.githubusercontent.com/22157215/27592513-f1ef4248-5b54-11e7-841b-203254e8be2c.jpg)

**Comment by astaric** [comment]
What would be the necessary steps to register as a publisher? Is it a free process, or does it involve a yearly fee (like Mac Developer program does)? Can Orange become a publisher or is it limited to only companies/institutions?

**Comment by jerneju** [comment]
It seems that is not free of charge (https://stackoverflow.com/questions/2660814/how-to-be-a-verified-publisher).

**Comment by janezd** [comment]
Not necessarily. We need to look into what kind of certificates can we obtain at our university or on Arnes. If Windows trusts them, they should be OK for signing.
